bridges/BAEBridge.php

```diff
@@ -79,7 +79,7 @@ public function getURI() {
 		return $uri;
 	}
 
-	private function remove_accents($string) {
+	private function removeAccents($string) {
```

Comment:
I think abetter name for this operation is `transliterate`. It also looks useful and re-useable elsewhere.